backends/bmv2/run-bmv2-test.py

```diff
@@ -48,7 +48,7 @@ def parse_args():
         "-bd",
         "--buildir",
         dest="builddir",
-        help="The path to the compiler build directory, default is \"build\".",
+        help="The path to the compiler build directory, default is current directory.",
```

Comment:
I have no problems with these changes, but I do not know whether they may break things for others.

Reply:
The point is to allow building in a build directory other than `build`, which used to work, but sometime in the past broke.